### PR TITLE
_posts/2022-03-08-PC-Engines-Firmware-v4.16.0.1: Add v4.16.0.1 releas…

### PR DESCRIPTION
**Pull request by IgorBagnucki**
…e letter

Signed-off-by: Igor Bagnucki <igor.bagnucki@3mdeb.com>

### CODE REVIEW

File: _posts/2022-03-08-PC-Engines-Firmware-v4.16.0.1.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+---
+
+# PC Engines apu coreboot Open Source Firmware v4.16.0.1
+
+## Key changes
+
+Mainline:
+
+### Changed
+- **Rebased** with official coreboot repository **commit** 36425312ee
```
Comment (artur-rs) [inline _posts/2022-03-08-PC-Engines-Firmware-v4.16.0.1.md:15]: @IgorBagnucki this commit is the same as in the rel 4.15.0.3

File: _posts/2022-03-08-PC-Engines-Firmware-v4.16.0.1.md [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+- Fixed the **hard disk** not **visible** in the Seabios Boot Menu
```
Comment (artur-rs) [inline _posts/2022-03-08-PC-Engines-Firmware-v4.16.0.1.md:16]: this change was introduced in the rel 4.15.0.3

**Review by artur-rs — CHANGES_REQUESTED** [review]

File: _posts/2022-03-08-PC-Engines-Firmware-v4.16.0.1.md (hunk 1 above)
Comment (IgorBagnucki) [reply _posts/2022-03-08-PC-Engines-Firmware-v4.16.0.1.md]: Thanks for catching that. Fixed

File: _posts/2022-03-08-PC-Engines-Firmware-v4.16.0.1.md (hunk 2 above)
Comment (IgorBagnucki) [reply _posts/2022-03-08-PC-Engines-Firmware-v4.16.0.1.md]: Fixed

**Review by miczyg1 — CHANGES_REQUESTED** [review]

File: _posts/2022-03-08-PC-Engines-Firmware-v4.16.0.1.md [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+
+## Key changes
+
+Mainline:
+
+### Changed
+- **Rebased** with official coreboot repository **commit** b4ba289fa5
+- Disabled **loglevel prefixes** introduced in coreboot 4.16
+- Disabled **ANSI escape** sequences introduced in coreboot 4.16
+
```
[the remark this answers is not in the document]
Comment (IgorBagnucki) [reply _posts/2022-03-08-PC-Engines-Firmware-v4.16.0.1.md:19]: Should be fixed now

**Review by miczyg1 — APPROVED** [review]

**Review by artur-rs — APPROVED** [review]

**Pull request merged**